### ISSUE TITLE
Execution hangs at "bash"

### ISSUE THREAD

**Issue by DigitalArcSystems** [asebak/raspberrypi-intellij#69]
First, I greatly enjoy this plugin.  One day my the plugin just stopped working for me.  I'm not sure what caused this, and I'm not sure how to debug it.  Here's the issue:

![createnewconfiguration](https://user-images.githubusercontent.com/197913/27541602-86b27aa2-5a52-11e7-9156-0ed3bf2217ce.png)

![configureandvalidate](https://user-images.githubusercontent.com/197913/27541608-8aadbc34-5a52-11e7-9555-aaec7d40590e.png)

![runwithnew](https://user-images.githubusercontent.com/197913/27541611-8d35521e-5a52-11e7-89a5-60fb07cf2161.png)

![endpoint](https://user-images.githubusercontent.com/197913/27541619-91600ec4-5a52-11e7-8805-acf3c7790693.png)

**Comment by asebak** [comment]
Hmm, not sure never seen this issue?  Do you still have this problem?

**Comment by mat0pad** [comment]
I just ran into this problem as well.....any updates?

**Comment by asebak** [comment]
What Operating system are you guys using?

**Comment by asebak** [comment]
@mat0pad which os were you guys using?

**Comment by mat0pad** [comment]
I'm on OS X Sierra

**Comment by asebak** [comment]
Yeah this is a mac issue, I don't have a mac so it's hard for me to test this out.
